src/plugins/intel_cpu/docs/debug_capabilities/infer_prc.md

```diff
@@ -0,0 +1,8 @@
+# Inference precision debug
+
+When developing/debugging accuracy issue related to inference precision feature, you can define following environment variables to control the type & number of nodes for which the specified inference precision is enforced:
+
+ - `OV_INFER_PRC_TYPES` : comma separated list of node types for which infer-precision is enforced
```

Comment:
Please align with other debug caps naming: should be `OV_CPU_INFER_PRC_TYPES` and `OV_CPU_INFER_PRC_CNT`

Reply:
OK